x-pack/plugins/security_solution/public/common/hooks/use_navigate_to_alerts_page_with_filters.ts

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { encode } from '@kbn/rison';
+
+import { SecurityPageName } from '../../../common/constants';
+import { formatPageFilterSearchParam } from '../../../common/utils/format_page_filter_search_param';
+import type { FilterItemObj } from '../components/filter_group/types';
+import { useNavigation } from '../lib/kibana';
+import { URL_PARAM_KEY } from './use_url_state';
+
+export const useNavigateToAlertsPageWithFilters = () => {
+  const { navigateTo } = useNavigation();
+
+  return (filterItems: FilterItemObj | FilterItemObj[]) => {
+    const url = encode(
```

Comment:
super nit: I would rename this constant to `urlFilterParams`

Reply:
changed!